fxa/tools/bearer.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+from __future__ import absolute_import
+from six.moves.urllib.parse import urlparse
+from fxa import core
+from fxa import oauth
+
+
+def get_bearer_token(email, password, scopes=None,
+                     account_server_url=None,
+                     oauth_server_url=None,
+                     client_id=None):
+
+    if not account_server_url and not oauth_server_url and not client_id:
```

Comment:
Should these conditions be `or`-ed rather than `and`-ed?  Or alternately, each one checked in its own statement with its own error message?